include/struct_decls/struct_plstring_decl.h

```diff
@@ -0,0 +1,8 @@
+#ifndef POKEPLATINUM_STRUCT_PLSTRING_DECL_H
+#define POKEPLATINUM_STRUCT_PLSTRING_DECL_H
+
+typedef u16 PLChar;
+
+typedef struct PLString_t PLString;
+
+#endif // POKEPLATINUM_STRUCT_PLSTRING_DECL_H
```

Comment:
A brief note, `struct_xxx_decl` headers were autogenerated with the intention that they are to be removed when dcoumenting. It might be too difficult to remove this header because it's referenced in too many places, but you could possibly replace the includes with includes to `plstring.h` and move the contents to `pl.string.h`.